PROTO_tests/tests/io_test.py

```diff
@@ -140,17 +141,32 @@ def make_multi_dtype_dict():
     }
 
 
-class TestParquet:
-    par_test_base_tmp = f"{os.getcwd()}/par_io_test"
+@pytest.fixture
+def par_test_base_tmp(request):
+    par_test_base_tmp = "{}/.df_test".format(os.getcwd())
     io_util.get_directory(par_test_base_tmp)
+
+    # Define a finalizer function for teardown
+    def finalizer():
+        # Clean up any resources if needed
+        io_util.delete_directory(par_test_base_tmp)
+
+    # Register the finalizer to ensure cleanup
+    request.addfinalizer(finalizer)
+    return par_test_base_tmp
+
+
+class TestParquet:
+    # par_test_base_tmp = f"{os.getcwd()}/par_io_test"
+    # io_util.get_directory(par_test_base_tmp)
```

Comment:
should we remove this commented out code?

Reply:
Removed.